src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -112,4 +139,22 @@ var fluid_2_0 = fluid_2_0 || {};
         that.events[that.model.isShowing ? "onPanelShow" : "onPanelHide"].fire();
     };
 
+    // panelId is passed in to ensure that it is evaluated before this
+    // function is called.
+    fluid.slidingPanel.setAriaProperties = function (that, panelId) {
+        that.locate("toggleButton").attr({
```

Comment:
This is a very ancient discussion, but I wonder if we ever came to a definitive conclusion about whether we should be using "attr" or "prop" for setting ARIA...... "properties" .... I imagine we did, but I don't seem to find a writeup anywhere. My sense is that it simply doesn't matter, since this presumably is one of that majority of properties that are "in 1-1 correspondence with attributes" but it would be good to be clear about this - especially since the name of this function is "setAriaProperties". It does seem that all the code I can find in trunk uses "attr" rather than "prop". We could even add a paragraph to the ILDH handbook here http://handbook.floeproject.org/index.php?title=WAI-ARIA_-_Beyond_semantic_tags about this